Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -0,0 +1,488 @@
+                                    "
+                                />
+                                <span class="error-message">
+                                    {{ errors[0] }}
+                                </span>
+                            </ValidationProvider>
+                        </Card>
+                        <Card
+                            title="1. Please confirm that you are over 12 years or older."
+                            additional-info="This citizen is 71 years old."
+                        >
+                            <ValidationProvider
+                                ref="confirmsOver12"
+                                v-slot="{ errors }"
+                                rules="oneOf:Yes,No"
+                                v-bind="$attrs"
+                                name="Confirms Over 12"
+                            >
+                                <OptionDetails
+                                    :value.sync="
+                                        covidTreatmentAssessmentRequest.confirmsOver12
+                                    "
+                                    :has-not-sure-option="false"
+                                    :has-additional-response="true"
+                                />
+                                <span class="error-message">
+                                    {{ errors[0] }}
+                                </span>
+                            </ValidationProvider>
+                        </Card>
+                        <Card
+                            title="2. Have you recently tested positive for COVID-19 in the last 7 days?"
+                        >
+                            <ValidationProvider
+                                ref="testedPositiveInPast7Days"
+                                v-slot="{ errors }"
+                                rules="oneOf:Yes,No,NotSure"
+                                v-bind="$attrs"
+                            >
+                                <OptionDetails
+                                    :value.sync="
+                                        covidTreatmentAssessmentRequest.testedPositiveInPast7Days
+                                    "
+                                    :has-additional-response="true"
+                                />
+                                <span class="error-message">
+                                    {{ errors[0] }}
+                                </span>
+                            </ValidationProvider>
+                        </Card>
+                        <Card
+                            title="3. Do you have any severe symptoms of COVID-19?"
+                        >
+                            <ValidationProvider
+                                ref="hasSevereCovid19Symptoms"
+                                v-slot="{ errors }"
+                                rules="oneOf:Yes,No,NotSure"
```

Comment:
We'll want to make sure that our validation rules match the expected radio buttons for each question, since they were changed.